indico/modules/events/papers/forms.py

```diff
@@ -35,16 +36,16 @@ def make_competences_form(event):
 
 
 class PaperTeamsForm(IndicoForm):
-    managers = PrincipalListField(_('Paper managers'), groups=True, event=lambda form: form.event,
-                                  description=_('List of users allowed to manage the call for papers'))
-    judges = PrincipalListField(_('Judges'),
-                                description=_('List of users allowed to judge papers'))
-    content_reviewers = PrincipalListField(_('Content reviewers'),
-                                           description=_('List of users allowed to review the content of the assigned '
-                                                         'papers'))
-    layout_reviewers = PrincipalListField(_('Layout reviewers'),
-                                          description=_('List of users allowed to review the layout of the assigned '
-                                                        'papers'))
+    managers = _LegacyPrincipalListField(_('Paper managers'), groups=True, event=lambda form: form.event,
```

Comment:
Same here; all cases where we use the `PrincipalListField` directly should be fine with the new field.

Reply:
Thanks.